pydoctor/test/test_commandline.py

```diff
@@ -1,10 +1,14 @@
+from contextlib import redirect_stdout
```

Comment:
pytest has the `capsys` fixture for capturing stdout/stderr. You can see examples of it in `pydoctor/test/test_epydoc2stan.py`.

Reply:
I will use the pytest helpers... but I think that the current code is much easier to read.

I have no idea what is going on in https://github.com/twisted/pydoctor/blob/master/pydoctor/test/__init__.py#L25

And this can it harder for people to contribute to a public project.

I am ok to use pytest and all kind of bleending edge libs for private projects, but I think that the public libs should be as accessible as possible.


... I know... I am getting older :)

Reply:
@mthuurne I have no idea what capsys is doing but I have lost a lot of time with it without any success so I went with simple redirect_stdout

Feel free to try to make the tests green using capsys ... some are working, other don't :(

Reply:
It has always worked well for me. I'll have a look.